### DISCUSSION TITLE
[Feature] Enable plural form at localizely

### DISCUSSION THREAD

**Discussion by waclaw66** [immich-app/immich#3851] [Feature Request]
### The feature

Instead of multiple keys for a single translated string, allow plurals for those strings.
E.g. album_thumbnail_card_item, album_thumbnail_card_items

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by daniele-athome** [comment]
Ref: https://localizely.com/language-plural-rules/
